src/containers/Forms/Payment/EditPaymentForm.js

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react';
+import { Dialog, DialogContent, Grid } from '@material-ui/core';
+import FormItem from '../../../components/UI/Form/FormItem/FormItem';
+import FormSubmission from '../../../components/UI/Form/FormSubmission/FormSubmission';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchPayments, updatePayment } from '../../../store/actions/paymentAction';
+import { getPaymentById } from './../../../store/actions/paymentAction';
+import { push } from 'connected-react-router';
+
+const EditPaymentForm = (props) => {
+  const selectedPayment = useSelector((state) => state.payments.payment);
+  const dispatch = useDispatch();
+  const paymentId = props.match.url.split('/')[3];
```

Comment:
есть  хуки для получения параметров, query параметров в  react-router ,  
import { useParams } from 'react-router';
```suggestion
 const { id: paymentId } = useParams();
```

Reply:
обновлено